cli/lib/kontena/plugin_manager.rb

```diff
@@ -2,27 +2,174 @@
 
 module Kontena
   class PluginManager
+
     include Singleton
 
     CLI_GEM = 'kontena-cli'.freeze
     MIN_CLI_VERSION = '0.15.99'.freeze
 
-    attr_reader :plugins
+    # Initialize plugin manager
+    def init
+      ENV["GEM_HOME"] = install_dir
+      Gem.paths = ENV
```

Comment:
GEM_HOME is the directory where gems will be installed to.
Gem.paths for some reason contains a dump of ENV. This reinitializes it after setting GEM_HOME.

```ruby
def self.paths=(env)
  clear_paths
  @paths = Gem::PathSupport.new env
  Gem::Specification.dirs = @paths.path # FIX: home is at end
end
```